docs/source/tutorials/automated-publication-with-ci.md

```diff
@@ -0,0 +1,160 @@
+
+```{code-block} toml
+:caption: hermes.toml
+:name: hermes.toml
+
+[harvest]
+from = [ "cff", "git" ]
+
+[harvest.cff]
+validate = false
+
+[deposit]
+mapping = "invenio"
+target = "invenio"
+
+[deposit.invenio]
+base_url = "https://sandbox.zenodo.org"
+```
+
+Copy this file to your repository and add it to version control:
+
+```bash
+git add hermes.toml
+git commit -m "Configure HERMES to harvest git and CFF, and deposit on Zenodo Sandbox"
+git push
+```
+
+## Get a personal access token for Zenodo Sandbox
+
+To allow GitHub Actions to publish our repository to Zenodo Sandbox for us,
+we need a personal access token from Zenodo Sandbox.
+
+Log in at https://sandbox.zenodo.org (you may have to register first),
+then [create a personal access token in your user profile](https://sandbox.zenodo.org/account/settings/applications/tokens/new/)
+with the scopes `deposit:actions` and `deposit:write`.
+
+Copy the newly created token into a new [GitHub Secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `ZENODO_SANDBOX` in your repository.
+
+## Configure a GitHub Action to automate publication 
+
+The HERMES project provides templates for continous integration systems in a dedicated repository:
+https://github.com/hermes-hmc/ci-templates.
+
+Copy the [template for GitHub to Zenodo Sandbox publication](https://github.com/hermes-hmc/ci-templates/blob/main/TEMPLATE_hermes_github_to_zenodo.yml)
+into the `.github/workflows/` directory in your repository, and rename it as you like (e.g. `hermes_github_to_zenodo.yml`).
+
+Go through the file, and look for comments marked with `# ADAPT`.
+Adapt the file to your needs.
+If you need help with how GitHub Action workflows work in general,
+have a look at the [documentation on GitHub](https://docs.github.com/actions).
+
+Add the workflow file to version control and push it.
+
+```{warning}
+If you haven't adapted the workflow file, and push it to the branch `main`, you will create an automatic publication at this point.
+```
+
+```bash
+git add .github/workflows/hermes_github_to_zenodo.yml
+git commit -m "Configure automatic publication with HERMES"
+git push
+```
+
+## Automatic publication workflow
+
+```{admonition} Congratulations!
+You can now automatically publish your repository to Zenodo Sandbox!
+```
+
+````{margin}
+```{mermaid}
+---
+---
+flowchart TD
+    t(("Trigger\n (i.e. push)")) -->
+    ci1("Run Harvest, Process,\n Curate steps") -->
+    pr1("Create Curation PR") -->
+    d{"Accept?"}
+    d -->|Merge| ci2("Run Deposit &\n Postprocess step") --> pr2("Create PR from\n Postprocessing") --> e((("End")))
+    d -->|Close| ci3("Cleanup") --> e((("End")))
+```
+````
+
+Now, whenever the GitHub Actions workflow is triggered, it will publish a new version of your repository to Zenodo Sandbox.
+If you haven't adapted the workflow file, this will happen whenever you push to your `main` branch.
```

Comment:
Again: No, it only creates the curation PR that needs to be merged before the deposition takes place.

Reply:
:+1: